keep/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+Keep acts as a single pane of glass and helps you turn 1000s into just 10s of meaningful alerts.
+
+We use monitors data, logs and events to correlate alerts and reduce noise.
+
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
+
+Users will:
+1. Be redirected to Keep's platform
+2. Click on the Datadog tile to initate an OAuth flow
+3. Review and confirm the required scopes in their Datadog account
```

Comment:
This should include the entire oauth flow a customer will take starting from the Datadog tile. Please include all steps a customer will take in the oauth flow